src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigWithFineTunedAlignments.java

```diff
@@ -118,66 +129,103 @@ public final boolean hasOnly2GoodAlignments() {
         return sourceTig.hasOnly2Alignments();
     }
 
-    public final boolean hasIncompletePicture() {
-        if ( hasOnly2GoodAlignments() )
-            return hasIncompletePictureFromTwoAlignments();
-        else
-            return hasIncompletePictureFromMultipleAlignments();
+    public enum AlignmentSignatureBasicTypes {
+        Suspicious, Simple, Complex;
+    }
+
+    public enum Suspicious {
+        Incomplete, Ambiguous, MisAssemblySuspect;
+    }
+
+    public AlignmentSignatureBasicTypes getAlignmentSignatureBasicTypes() {
+        if ( hasEquallyGoodAlnConfigurations || hasIncompletePicture() || (! isInformative()) ) {
+            return AlignmentSignatureBasicTypes.Suspicious;
+        } else if ( hasOnly2GoodAlignments() ) {
+            return AlignmentSignatureBasicTypes.Simple;
+        } else {
+            return AlignmentSignatureBasicTypes.Complex;
+        }
+    }
+
+    public Suspicious getReasonForSuspicion() {
+
+        if ( hasEquallyGoodAlnConfigurations ) { // ambiguous
+            return Suspicious.Ambiguous;
+        } else if ( hasIncompletePicture() ) {
+            return Suspicious.Incomplete;
+        } else if ( ! isInformative() ) { // un-informative contig, mis-assembly suspect
```

Comment:
Why does the fact that the contig has only one alignment mean that it's a mis-assembly suspect? I don't understand the logical leap there.

Reply:
After thinking about this a little more I think I'd just recommend calling them `UNINFORMATIVE` rather than trying to call out the assembly or the mapping. Does that work for you?

Reply:
That's better! Thanks!